test_runner/src/main/kotlin/ftl/args/AndroidArgs.kt

```diff
@@ -85,3 +85,9 @@ val AndroidArgs.isInstrumentationTest
 val AndroidArgs.isRoboTest
     get() = appApk != null &&
             (roboDirectives.isNotEmpty() || roboScript != null)
+
+val AndroidArgs.isSanityRobo
+    get() = appApk != null &&
```

Comment:
isNotNull?

Reply:
Good point. I followed the same convention that was used in older code in this file, but you are right and I'll change it there as well